CIScripts/Test/Tests/Protocol/SingleHostWithUtils.Tests.ps1

```diff
@@ -166,6 +171,16 @@ Describe "Single compute node protocol tests with utils" {
         $SystemConfig = Read-SystemConfig -Path $TestenvConfFile
         $IisTcpTestDockerImage = "iis-tcptest"
 
+        [Diagnostics.CodeAnalysis.SuppressMessageAttribute("PSUseDeclaredVarsMoreThanAssignments",
+            "AgentLogs", Justification="Analyzer doesn't understand relation of Pester blocks"
+        )]
+        $AgentLogs = New-LogSource -Sessions $Session -Path "C:/ProgramData/Contrail/var/log/contrail/*.log"
+        [Diagnostics.CodeAnalysis.SuppressMessageAttribute("PSUseDeclaredVarsMoreThanAssignments",
+            "DriverLogs", Justification="Analyzer doesn't understand relation of Pester blocks"
+        )]
+        $DriverLogs = New-LogSource -Sessions $Session -Path "C:/ProgramData/ContrailDockerDriver/log.txt"
```

Comment:
If we move those variable initialization close to the place, where variables are actually used, we might get rid of static analysis message suppression.